bundles/org.openhab.binding.shelly/doc/images/UseCaseSmartRoller.md

```diff
@@ -0,0 +1,225 @@
+- get status updates from roller operations by observing channel 'roller#state' and 'roller#stopReason', those report an updated status once the operation is completed
+- select a position by sending a number value of 100..0 to channel 'roller#rollerpos'
+- or use the defined favorites by sending favorite id to channel 'roller#rollerFav'
+- you might set a auto-timer, e.g. one the roller is opened close it again after xx minutes by sending a value to the 'roller#autoOn' or 'roller#autoOff' channels
+- you could also get the position of the roller from 'roller#control' (by linking a Number item) or the last status from 'roller#state'.
+Please note that the device only provides update after the roller stops, not during movement so it's not possible to get position updates while the roller is moving.
+- of course you could time-triggered rules, trigger the shutter depending on sunrise/sunset within openHAB, but also directly on the device
+
+## Extending the Use Case
+
+There are two valuable options, which even extend and improve the user experience.
+
+- Button-1
+
+Using the described setup you could add a Button-1 to trigger rules, e.g.
+- 1xshort: open roller
+- 2xshort: go to 25% (because disturbs enjoying your morning coffee with your wife)
+- 3xshort: go to 75% (sun is disturbing while watching your TV)
+- long   : close shutter (if not done by some automation)
+
+- Voice control
+And of course you could utilize openHAB's voice integration, e.g. with Alexa, Siri or Google Home (who need's Cortana).
+
+- Scenes
+There are more of less limits when integrating even more advanced roller control based on openHAB rules. 
+This allows to provide a scene to the user, which does a lot of complex stuff under the hood.
+
+Be welcome to contribute your ideas to the community, this is what creates the difference between openHAB/Shelly setup compared to rollers with button control.
+
+### Resources
+
+There is a bunch of ressources you could browse around openHAB, Shelly or the binding
+- Current builds of the binding:
+Latest **DEV** build: [2.5.11](https://github.com/markus7017/myfiles/blob/master/shelly/org.openhab.binding.shelly-2.5.11-SNAPSHOT.jar?raw=true) - [3.0.0](https://github.com/markus7017/myfiles/blob/master/shelly/org.openhab.binding.shelly-3.0.0-SNAPSHOT.jar?raw=true) - [README](https://github.com/markus7017/myfiles/blob/master/shelly/README.md) -  [Installation](https://github.com/markus7017/myfiles/blob/master/shelly/READMEbeta.md) - [Bugs/Features](https://github.com/openhab/openhab-addons/issues?q=is%3Aissue+is%3Aopen+%5Bshelly%5D) - [Firmware Index](https://repo.shelly.cloud/files/firmware/?fbclid=IwAR0Kz6bjMGoq_HYU85QAdl8b-xh7E3IPa4ENKjqsvmMAYQ6QGC8ODvqPzUs) - [Firmware Archive](http://archive.shelly-faq.de) - [API Doc](https://shelly-api-docs.shelly.cloud/?fbclid=IwAR23ukCi_3aBSTPRHYUIcpr0pLi0vcyL0fF0PnJQdFvkkc8_Zo5LkAcli_A#http-server)
```

Comment:
Can you put each in a separate line?

Reply:
changed, I removed all links to my personal repo